timescaledb/how-to-guides/compression/about-compression.md

```diff
@@ -218,7 +220,7 @@ should be `device_id, metric_name`:
 The `segmentby` columns are useful, but can be overused. If you specify a lot of
 `segmentby` columns, the number of items in each compressed column is reduced,
 and compression is not as effective. A good guide is for each segment to contain
-at least 100 rows per chunk. To achieve this, you might also need to use  
+at least 100 rows per chunk. To achieve this, you might also need to use
 the `compress_orderby` column.
```

Comment:
Hmm i think this could be more verbose to make it more clear. compress_orderby has no influence on segment size. The advice we want to give here is to not add to many columns as segmentby

Reply:
What would you suggest we put there, @svenklemm ?